main.tf

```diff
@@ -112,8 +112,12 @@ resource "aws_msk_cluster" "default" {
   enhanced_monitoring    = var.enhanced_monitoring
 
   broker_node_group_info {
-    instance_type   = var.broker_instance_type
-    ebs_volume_size = var.broker_volume_size
+    instance_type = var.broker_instance_type
+    storage_info {
+      ebs_storage_info {
```

Comment:
What aws version is required to use this new block argument? Can you set the minimum aws version in versions.tf and examples/complete/versions.tf to make sure this new argument will work for all users?